src/location.cpp

```diff
@@ -150,12 +28,19 @@ bool is_github_archive(std::string_view s) { return is_github(s) && is_archive(s
 
 bool is_url(std::string_view l) { return curl::is_url(l); }
 
-bool is_github_org_repo_shorthand(std::string_view s) { return github::is_shorthand(s); }
+bool is_github_shorthand(std::string_view s) { return github::is_shorthand(s); }
 
 bool is_github_repo(std::string_view s) { return is_github(s) && !is_archive(s); }
 
 bool is_reachable(std::string_view l) {
-   if (is_github_repo(l) || is_github_org_repo_shorthand(l)) {
+   if (!is_github_shorthand(l)) {
+      system::error_log("In this version of antler-proj only github shorthands are supported. Generalized git repos and archives will be supported in the next version.");
```

Comment:
Maybe we want to under promise and over deliver here?
```suggestion
      system::error_log("In this version of antler-proj only github shorthands are supported. Generalized git repos and archives will be supported in a future version.");
```

Reply:
fixed in 8c9b5c7